item-search/README.md

```diff
@@ -55,8 +55,11 @@ The following Link relations shall exist in the Landing Page (root).
 | -------- | --------- | ---------------------- | --------------------------- |
 | `search` | `/search` | STAC API - Item Search | URI for the Search endpoint |
 
-The `search` link relation shall have a `type` of `application/geo+json` and a `method` of `GET`, and may also
-a link with a `method` of `POST` if the server supports it.
+This `search` link relation shall have a `type` of `application/geo+json`. If no `method` attribute is
```

Comment:
I think it only SHOULD have it or let's say the default for search should probably be `application/geo+json`. We do the same with most other links, e.g. child, item, ...

Reply:
And a general observation, but I think we mostly use MUST instead of SHALL in the documentation. Your recent PRs use SHALL and I think we should be more consistent here. Could we try to use MUST instead of SHALL? I just had to look it up again whether SHALL was equal to MUST or SHOULD and found it confusing.

Reply:
Filed https://github.com/radiantearth/stac-api-spec/issues/275 to discuss